app/src/main/java/com/innowisegroup/datetimepicker/MainActivity.kt

```diff
@@ -10,16 +10,11 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
-        val dateTimePickerDialog = DateTimePickerDialog(
-                null,
-                null,
-                null,
-                wrapSelectionWheel = false,
-                withOnlyDatePicker = false,
-                withOnlyTimePicker = false)
+        val pickerDialog = PickerDialog.createDateTimePickerDialog()
+
         val btn = findViewById<Button>(R.id.btn)
         btn.setOnClickListener {
-            dateTimePickerDialog.showDialog(supportFragmentManager)
+            pickerDialog.showDialog(supportFragmentManager)
```

Comment:
So is it "nice to call" API - PickerDialog.createDateTimePickerDialog().showDialog(supportFragmentManager)
Hmm...Every word ends with ...dialog

1. Probably, let's rename it to ReelPicker and function calls too (See below)
2. also rename FragmentDatePicker -> DatePickerFragment and FragmentTimePicker -> TimePickerFragment (in separate commit)

ReelPicker.createDateDialog().show()
ReelPicker.createTimeDialog().show()
ReelPicker.createDateTimeDialog().show()